src/Compatibility/Microsoft.DotNet.Compatibility/Microsoft.DotNet.Compatibility.csproj

```diff
@@ -0,0 +1,87 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net472</TargetFrameworks>
+    <IsPackable>true</IsPackable>
+    <IsShippingPackage>true</IsShippingPackage>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Why do we need a Version prefix?

Reply:
This is how arcade versioning works. If the build is stable this would be treated as the package version, if not the suffix is added.

Reply:
Its basically use the sdk convention to override the default 6.0 default branding